addons/website_blog/controllers/main.py

```diff
@@ -62,6 +63,9 @@ def nav_list(self, blog=None):
             group['date_end'] = '%s' % datetime.date.strftime(end_date, tools.DEFAULT_SERVER_DATE_FORMAT)
         return groups
 
+    def has_author_avatar(self, author_id):
```

Comment:
remove this method, and don't give it in values to render qweb template. In template, test if the related field `avatar_author` is false.